src/resources/base.py

```diff
@@ -0,0 +1,19 @@
+from rich.text import Text
+
+
+class AbstractDungeonEntity:
```

Comment:
consider using python abstract class. see https://docs.python.org/3/library/abc.html

Reply:
The file and the class should have the same name.

Reply:
Fixed